.github/workflows/on-push.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Tests and Linting
+
+on:
+  push
+
+jobs:
+  deploy:
+    name: Run Tests and Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Ref
+        uses: actions/checkout@v1
+        with: 
+          ref: ${{ github.event.push.after }}
+      - name: Run Tests and Lint
+        uses: evolv-ai/github-actions/cicd/nodejs13@v1
```

Comment:
The `push` trigger feels like it relates to actions that are more repo-dependent than shared. So I would say just use normal `run` directives here to keep it local. Plus I also don't think the shared action does anything besides running the _test.sh_ that you've defined in this repo anyway.

```
    - run: |
        npm ci --production=false
    - run: |
        npm test
    - run: |
        npm run lint
```

Reply:
Do we have access to the correct node version if we do that though?

Reply:
The action that i have it running in is using Node 13

Reply:
There's a setup action

```
- uses: actions/setup-node@v1
  with:
    node-version: '13.0.0'
```